processes_notwindows.go

```diff
@@ -76,7 +76,13 @@ func processesFromProc() ([]ProcStat, error) {
 			log.Errorf("Failed to read cmdline(%s): %s", stats[0], err.Error())
 		}
 
-		stat := ProcStat{PID: pid, Name: string(bytes.TrimRight(comm, "\n")), Cmdline: strings.Replace(string(bytes.TrimRight(cmdline, "\x00")), "\x00", " ", -1)}
+		ppid, err := strconv.Atoi(string(stats[4]))
+
+		if err != nil {
+			log.Errorf("Failed to convert PPID(%s) to int: %s", stats[4], err.Error())
+		}
+
+		stat := ProcStat{PID: pid, ParentPID: ppid, Name: string(bytes.TrimRight(comm, "\n")), Cmdline: strings.Replace(string(bytes.TrimRight(cmdline, "\x00")), "\x00", " ", -1)}
```

Comment:
To make it easier to read I would suggest to create `ProcState` in multiple lines and not one.